packages/gpm.rb

```diff
@@ -0,0 +1,82 @@
+require 'package'
+
+class Gpm < Package
+  description 'A mouse server for the console and xterm'
+  homepage 'https://www.nico.schottelius.org/software/gpm/'
+  version '1.20.7-e82d'
```

Comment:
No `@_ver`?

Reply:
Not useful until the next tagged commit, right?

Reply:
Wouldn't this be the "correct" way to do it?
```
@_ver = '1.20.7'
version "#{@_ver}-e82d"
```